docs/user-guide/configuring-at-tls-for-zowe-server.md

```diff
@@ -684,7 +684,3 @@ TTLSCipherParms                   CipherParms
 ```
 
 </details>
-
-## Additional Zowe feature configuration with AT-TLS
-
-The Zowe Application Framework also leverages AT-TLS. For more information, see [Using AT-TLS in the App Framework](../user-guide/mvd-configuration#using-at-tls-in-the-app-framework).
```

Comment:
removed this because not only does this header no longer exist, but that page referenced now just points back to this page regarding at-tls because this page is now the authority on at-tls.